composer.json

```diff
@@ -111,7 +111,7 @@
     "phpunit/phpunit": "^7.0",
     "presta/sitemap-bundle": "^1.5.3",
     "prezent/doctrine-translatable": "^1.2",
-    "prezent/doctrine-translatable-bundle": "^1.0.3",
+    "prezent/doctrine-translatable-bundle": "1.0.3",
```

Comment:
What was the error and in which version of prezent/doctrine-translatable-bundle? You should definitely elaborate in the commit message.

Reply:
You do not need to use this anymore. This problem has been already fixed. Please remove this commit.

Reply:
prezent released new version with fix of that error, ill drop this commit.